Makefile

```diff
@@ -21,6 +21,9 @@ IMAGE_VERSION ?= v0.7.0
 # Use a custom version for E2E tests if we are in Prow
 ifdef AZURE_CREDENTIALS
 override IMAGE_VERSION := e2e-$(GIT_COMMIT)
+ifdef TEST_WINDOWS
```

Comment:
It seems that this judgment does not take effect in ci. I think it should be `ifdef $TEST_WINDOWS`.